src/Pos/Update/Poll/Pure.hs

```diff
@@ -0,0 +1,70 @@
+-- | Pure Poll
+
+module Pos.Update.Poll.Pure
+       ( PurePoll (..)
+       ) where
+
+import           Control.Lens               (to, (%=), (.=))
+import qualified Data.HashMap.Strict        as HM
+import qualified Data.HashSet               as HS
+import           System.Wlog                (CanLog, HasLoggerName, NamedPureLogger)
+import           Universum                  hiding (toList)
```

Comment:
`Universum` should be first line in import section in separate block according to style-guide. Also, why `toList` is hidden?